admin_manual/configuration/server/config_sample_php_parameters.rst

```diff
@@ -722,32 +722,38 @@ The default value is ``ownCloud``.
 			'shared_secret' => '57b58edb6637fe3059b3595cf9c41b9',
 			'users' => ['user1'],
 			'apps' => ['files_texteditor'],
-			'logfile' => '/tmp/test.log'
+			'logfile' => '/tmp/test1.log'
 	        ],
 	        [
 			'shared_secret' => '57b58edb6637fe3059b3595cf9c41b9',
-			'users' => ['user1'],
+			'users' => ['user2'],
 			'apps' => ['gallery'],
-			'logfile' => '/tmp/gallery.log'
+			'logfile' => '/tmp/test2.log'
 	        ],
 	],
 
-Log condition for log level increase based on conditions. Once one of these
-conditions is met, the required log level is set to debug. This allows to
-debug specific requests, users or apps
+Log condition for log level increase based on conditions. 
+You can populate one, two or all three conditions.
+The first met condition inside a condition block sets the log level to debug and writes a log entry if avaliable.
+This allows to debug specific requests, users or apps.
 
 Supported conditions:
- - ``shared_secret``: if a request parameter with the name `log_secret` is set to
-               this value the condition is met
- - ``users``:  if the current request is done by one of the specified users,
-               this condition is met
- - ``apps``:   if the log message is invoked by one of the specified apps,
-               this condition is met
- - ``logfile``: the log message invoked by the specified apps get redirected to
-	   this logfile, this condition is met
-	   Note: Not applicable when using syslog.
-
-Defaults to an empty array.
+ - ``shared_secret``: If a request parameter with the name ``log_secret`` is set to
+               the value of this ``shared_secret``, the condition is met.
```

Comment:
s/is set to the value of this/is set to/g

Reply:
I wanted to explicitly highlight that the "value" of shared_secret and log_secret must match.
You have a config.php parameter and you have a parameter named different used at another place but both "values" must match.